File: Tugas B/cpp/memory.cpp
#include <iostream>
#include <string>

using namespace std;

class memory{
	private:
		string freq;
        string size;
        string cuda;
	
	public:
		//Construct
		memory(){}
		
		memory(string freq, string size, string cuda){
			this->freq = freq;
			this->size = size;
			this->cuda = cuda;
		}
		
		//Set dan Get
		void setfreq(string freq)
        {
            this->freq = freq;
        }
		string getfreq(){
			return this->freq;
		}
		
		void setsize(string size){
			this->size = size;
		}
		string getsize(){
			return this->size;
		}
		
		void setcuda(string cuda){
			this->cuda = cuda;
		}
		string getcuda(){
			return this->cuda;
		}
			
		
		~memory(){
		}
};

File: Tugas B/php/memory.php
<?php

class memory
{
	
	private $freq = "";
	private $size = "";
	private $cuda = "";
	
	
	//Construct
	public function __construct($freq = "", $size = "", $cuda = ""){
		$this->freq = $freq;
		$this->size = $size;
		$this->cuda = $cuda;
	}
	
	//Set dan get
	public function setfreq($freq){
		$this->freq = $freq;
	}
	public function getfreq(){
		return $this->freq;
	}
	
	public function setsize($size){
		$this->size = $size;
	}
	public function getsize(){
		return $this->size;
	}
	
	public function setcuda($cuda){
		$this->cuda = $cuda;
	}
	public function getcuda(){
		return $this->cuda;
	}
}
?>